inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/consts/SinkType.java

```diff
@@ -36,5 +36,6 @@ public class SinkType {
     public static final String ORACLE = "ORACLE";
     public static final String TDSQLPOSTGRESQL = "TDSQLPOSTGRESQL";
     public static final String DLCICEBERG = "DLCICEBERG";
+    public static final String BASE = "BASE";
```

Comment:
Excuse me, what is the `BASE` type?
I know `MYSQL`, `ORACLE`, `HIVE`, etc., but I haven't heard of the `BASE` type.

Reply:
To differ from sink type in Sort-flink.  
Base is just a prefix,  the real type is base_kafka, base_pulsar, etc.
The Base_XXX type means all fields are directly inserted into the very table, instead of complex checking and multi-table operation

Reply:
has renamed to **custom**